README.md

```diff
@@ -42,7 +42,9 @@ Please see [The official documentation](https://www.vaultproject.io/docs/config/
 
 * `purge_config_dir`: Whether the `config_dir` should be purged before installing the generated config.
 
-* `install method`: Supports the values `repo` or `archive`. See [Installation parameters](#installation-parameters).
+* `install_method`: Supports the values `repo` or `archive`. See [Installation parameters](#installation-parameters).
+
+* `manage_unzip`: Whether to manage the `unzip` package on Debian/Ubuntu systems. (defaults to true if `install_method` is set to `archive`)
```

Comment:
A PR has been opened about this in the past: https://github.com/jsok/puppet-vault/pull/60

I'm still inclined to exclude this functionality, but we should probably mention that it's the users responsibility to manage the package(s) when using the `archive` `install_method`.